yellowbrick/text/dispersion.py

```diff
@@ -95,34 +120,91 @@ def fit(self, text):
         text : list
```

Comment:
For our other text visualizers, we allow users to pass data to `fit()` as either an ndarray or a dataframe, so we may want to anticipate the case that a user passes in the corpus as a dataframe here,

Reply:
Are you referring to the `y` variable?

Reply:
I was referring to the `text` (i.e. the corpus data), which is passed to `fit()`.

Reply:
duh! 👎 FAIL

Reply:
How would the dataframe be structured?
rows = documents
columns = words